docs/DeveloperGuide.md

```diff
@@ -1018,8 +1018,8 @@ Similar to **UC03 Delete a contact**, except,
       1. `12345678`
    1. Invalid `Phone` fields:
       1. `+65 12345678`
-      2. `01234-012345`
-      3. `(01234) 012345` 
+      1. `01234-012345`
+      1. `(01234) 012345` 
 
    We plan to update the `VALIDATION_REGEX` to allow other possible phone numbers including, but not limited to, those under "Invalid `Phone` fields" above, as well as limiting the length of a valid `Phone` field to follow the longest conventionally accepted phone number.
 1. The current restrictions allow for adding a `Project` with an overdue `ProjectDeadline`, allowing users to add projects with deadlines that are already overdue. We plan on adding further checks when setting the `ProjectDeadline` field to prevent users from setting an overdue deadline, and show an error message highlighting the reason for the restriction: `The project deadline could not be set as it is already past the deadline.` 
```

Comment:
Can't seem to tag the line below, but do consider suggesting some enhancements to handle possible reported feature flaws to UI; for example could suggest some enhancements such that there is a limit to displayed length of certain fields so that tags are more visible to users in such cases.

Else, feel free to remove the placeholder line below